test/sql/index/art/test_art_index_join.test

```diff
@@ -0,0 +1,33 @@
+# name: test/sql/index/art/test_art_index_join.test
+# description: ART Join
+# group: [art]
+
+statement ok
+CREATE TABLE Person (id bigint PRIMARY KEY);
+
+statement ok
+CREATE TABLE Person_knows_Person (Person1id bigint, Person2id bigint);
+
+statement ok
+INSERT INTO Person VALUES (1), (2), (3), (4), (5);
+
+statement ok
+INSERT INTO Person_knows_Person VALUES (1, 2), (1, 3), (1, 4), (2, 3), (3, 4), (4, 5);
```

Comment:
Could we also add some tests that perform the index join while e.g. appends or updates are performed on the indexed columns? These could be added in `test/sql/parallelism/interquery/test_concurrent_index.cpp`.